debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -34,6 +34,14 @@
 
 CLimits.USHRT_MAX = 0xFFFF
 
+/* The below two fields match the current flags in the internal
+ * builds. If the OpenJ9 MethodHandles are disabled and OpenJDK
+ * MethodHandles are enabled in the internal builds, then they
+ * should be updated as follows:
+ *    J9BuildFlags.J9VM_OPT_METHOD_HANDLE = 0
+ *    J9BuildFlags.J9VM_OPT_OPENJDK_METHODHANDLE = 1
+ */
+J9BuildFlags.J9VM_OPT_METHOD_HANDLE = 1
```

Comment:
Even after declaring `J9BuildFlags.J9VM_OPT_METHOD_HANDLE` here, the below error persists in the internal builds:
```
    [javac] /tmp/bld_66318/job_78749901/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/RomClassWalker.java:781: error: cannot find symbol
    [javac] 		if (J9BuildFlags.J9VM_OPT_METHOD_HANDLE && !J9BuildFlags.J9VM_OPT_OPENJDK_METHODHANDLE) {
    [javac] 		                ^
    [javac]   symbol:   variable J9VM_OPT_METHOD_HANDLE
    [javac]   location: class J9BuildFlags
```
Is there an internal dependency where `J9BuildFlags.J9VM_OPT_METHOD_HANDLE` needs to be defined? or are the builds dependent on the previous build i.e. do we need to push the [CompatibilityConstants29.dat](https://github.com/eclipse-openj9/openj9/pull/19075/files#diff-b8df33389a91caacde9d5fb83e955d0b09e9b65f58bcb096ad1391e1e0bdbaa2) change separately so that it is recognized in future builds?

Reply:
No, there should be no dependency on earlier builds for entries in `CompatibilityConstants29.dat`.